### ISSUE TITLE
Bypassing chest logs exploit.

### ISSUE THREAD

**Issue by andrewkm** [LogBlock/LogBlock#759]
One of my upper staff recently reported this and its quite concerning:

Sending this per your interest expressed in the Discord DM re the wolf situation:

Basically, when editing a chest, logblock only logs the user who closes the chest (and only logs anything once that happens). This means that if I, along with another user, were to both open a chest, I put in a bunch of blocks, the other user takes them and exits the chest interface (Not closes it - they can't close when it when I'm still in there) and [I]then[/I] I close the chest, then there won't be any record of movement at all (or just that of me putting blocks in, not the other user taking them out).

Here's a video of how it's done to explain what can be hard to convey over text:

https://www.youtube.com/watch?v=71MM3MS2WTo

TL;DR, it's totally possible to edit chests with the assistance of another user without any trace of it being down. Not[I] too[/I] bad, but this is still potentially problematic in cases of scamming duos, esp with one in vanish and he other providing the "service" (ie creating a chest to facilitate a transaction, smelt, etc., the person providing the service TPs a user in vanish, they both open the chest, the vanished user steals it and there's no trace) - Which things like these are one of the primary reasons why the /own command (You'd have scamming duos who would pick up tools being given for an EREP in those cases).

**Comment by Brokkonaut** [comment]
Yes I know (https://github.com/LogBlock/LogBlock/issues/565) - The problem is, that this is not easy to fix, because there are a lot of inventory actions that hat to be considered for a proper logging.

**Comment by Onyxtite** [comment]
This seems an important issue, but i can see a posible workarround that might be easy to code. Disable multiple players from interacting with a chest at the same time: I mean, more complex solutions would be better but this should work and be relatively simple to implement. Just an idea.

**Comment by Brokkonaut** [comment]
I will not add that to logblock because it might be confusing for many users. but it should be easy to create a seperate plugin for that.

**Comment by andrewkm** [comment]
Thanks for the responses :)
